src/wp-includes/vars.php

```diff
@@ -126,6 +126,12 @@
  */
 $is_nginx = ( str_contains( $_SERVER['SERVER_SOFTWARE'], 'nginx' ) );
 
+/**
+ * Whether the server software is Caddy or something else
+ * @global bool $is_caddy
```

Comment:
```suggestion
 * Whether the server software is Caddy or something else
 *
 * @global bool $is_caddy
```

Reply:
The end period is missing in all short description.

Reply:
Better to handle that in a separate commit